### PR TITLE
add payroll deductions

### PR DESCRIPTION
**Pull request by alex-marie-merino**

### CODE REVIEW

File: src/api-routes/servicenow/employees/{osuId}/deductions.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+
+import { errorHandler, errorBuilder } from 'errors/errors';
+import { postDeductions } from '../../../../db/oracledb/servicenow-dao';
+import { serializeDeductions } from '../../../../serializers/deductions-serializer';
```
Comment (Ruefa) [inline src/api-routes/servicenow/employees/{osuId}/deductions.js]: Dont need the `../` should work with just `db/oracledb/servicenow-dao`

File: src/db/oracledb/servicenow-dao.js [hunk 2]
```diff
@@ -353,11 +353,43 @@ const postJob = async (body) => {
...
+      positionNumber: attributes.job.positionNumber,
+      suffix: attributes.job.suffix,
+      startDate: attributes.job.startDate,
+      result: { type: STRING, dir: BIND_OUT, maxSize: 4000 },
+    });
+
+    if (_.startsWith(result, '**NOT FOUND**')) {
+      throw createError(400, result);
+    }
+    await connection.commit();
```
Comment (Ruefa) [inline src/db/oracledb/servicenow-dao.js:380]: Need to check the result of the commit to make sure it was successful

**Review by Ruefa — CHANGES_REQUESTED** [review]
Overall looks good. Needs a few updates and the new contrib commit

File: src/db/oracledb/servicenow-dao.js (hunk 2 above)
Comment (alex-marie-merino) [reply src/db/oracledb/servicenow-dao.js:380]: Do you have an example of this? I was just copying the syntax Tso-Liang used for the other functions.

Comment (Ruefa) [reply src/db/oracledb/servicenow-dao.js:380]: You're right. Its probably fine how it is then.

File: src/api-routes/servicenow/employees/{osuId}/deductions.js (hunk 1 above)
Comment (alex-marie-merino) [reply src/api-routes/servicenow/employees/{osuId}/deductions.js]: Fixed

**Review by Ruefa — APPROVED** [review]

**Pull request merged**